internal/api/migrate/mapping.cuepart

```diff
@@ -165,7 +165,12 @@ spec: {
                 if grafanaVar.sort != _|_ if #mapping.sort[grafanaVar.sort] != _|_ {
                     sort: #mapping.sort[grafanaVar.sort]
                 }
-                // defaultValue: nothing to map to this field
+                if grafanaVar.current != _|_ if grafanaVar.current.value != _|_ {
+                    defaultValue: grafanaVar.current.value
+                }
+                if (*(grafanaVar.current.value) | _|_ ) == _|_ if *(grafanaVar.includeAll) | false  {
```

Comment:
oh interesting I never thought of this possibility to check the existence of nested fields directly

Reply:
But if there is no current value defined on Grafana side I think we shouldnt set defaultValue then?

Reply:
Idea is to "mimic" Grafana default behaviour, if current is not set at all, 'All is used by default; when dashboard is loaded with includeAll: true. 

Without explicitly setting defaultValue: `$__all` in here,  Perses sets first value of the list as the defaultValue instead when dashboard is loaded. 

I think using `All` is better default when current is not set.

Reply:
Okay so actually on our side we have some issues with this behavior of grafana, it doesn't scale well for variables that return a lot of values, on the initial load of the dashboard if no defaultValue is defined then the fact to take All by default requires a lot of time to compute & load the dashboard, and this for any user, even the ones only interested in visualizing a subset of objects.

If you want All to be the default you can always save the dashboard with this explicit default value set, and this is already covered by your previous conditional statement here.

So yeah, we're not really in favor of the proposed migration logic here. And if ever this should change in the future, it should be done in the Perses's code responsible of that behavior: https://github.com/perses/perses/blob/e2c3831331c62a01947791e1d576d2b5dfea6807/ui/dashboards/src/context/VariableProvider/hydrationUtils.ts#L32-L39